src/components/CustomModal/CustomModal.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { View, Text } from 'react-native';
+import CustomButton from 'src/components/CustomButton';
+import styles from './styles';
+import { simpleText } from 'src/style/commonStyles';
+
+const CustomModal = ({
```

Comment:
Also we have [RN Modal](https://facebook.github.io/react-native/docs/modal).

And there is an interesting video with example how to add interesting animation to your modal:
[https://www.youtube.com/watch?v=MfpEROHbnTA&t=1s](https://www.youtube.com/watch?v=MfpEROHbnTA&t=1s)